lib/rihanna/job.ex

```diff
@@ -94,8 +94,7 @@ defmodule Rihanna.Job do
     now = DateTime.utc_now()
 
     result =
-      Postgrex.query(
-        Rihanna.Job.Postgrex,
+      enqueue_conn_exec(
```

Comment:
call it `producer_query`

to differentiate from `consumer_query` which requires the locking.